tests/PHPUnit/System/expected/test_TwoVisitors_twoWebsites_differentDays_VisitsSummary.get_firstSite_lastN__API.getProcessedReport_day.xml

```diff
@@ -44,10 +44,10 @@
 	</columns>
 	<reportData>
 		<result prettyDate="Sunday 3 January 2010">
-			<nb_uniq_visitors>1</nb_uniq_visitors>
+			<nb_uniq_visitors>2</nb_uniq_visitors>
 			<nb_users>0</nb_users>
-			<nb_visits>1</nb_visits>
-			<nb_actions>1</nb_actions>
+			<nb_visits>2</nb_visits>
+			<nb_actions>2</nb_actions>
 			<max_actions>1</max_actions>
 			<nb_actions_per_visit>1</nb_actions_per_visit>
 			<avg_time_on_site>00:00:00</avg_time_on_site>
```

Comment:
changes in this test output seems strange to me, maybe something wrong, need to check tomorrow